pkg/scraping/scraper.go

```diff
@@ -108,11 +111,27 @@ func (s *Scraper) queryNode(ctx context.Context, url string, events chan entitie
 		return
 	}
 	events <- entities.NewHeightEvent(url, ts, v, h)
+
+	bs, err := node.baseTarget(h)
+	if err != nil {
+		nodeUrl, err := u.Parse(node.cl.GetOptions().BaseUrl)
+		if err != nil {
+			log.Printf("Failed to parse node url: %v", err)
+			return
```

Comment:
Actually, I think that `queryNode` method should return error and that error should be logged by the caller.

Reply:
it is logged and an alert is generated